modules/sdk-test/src/bitgo/lib/test_bitgo.ts

```diff
@@ -0,0 +1,624 @@
+//
```

Comment:
we should rename this file to something different since it no longer exports TestBitgo.